docs/light/getting_started.md

```diff
@@ -1,40 +1,38 @@
 # Getting Started
 
 To start a rest server, we need to specify the following parameters:
-| Parameter   | Type      | Default                 | Required | Description                                          |
-| ----------- | --------- | ----------------------- | -------- | ---------------------------------------------------- |
-| chain-id    | string    | null                    | true     | chain id of the full node to connect                 |
-| node        | URL       | "tcp://localhost:46657" | true     | address of the full node to connect                  |
-| laddr       | URL       | "tcp://localhost:1317"  | true     | address to run the rest server on                    |
-| trust-node  | bool      | "false"                 | true     | Whether this LCD is connected to a trusted full node |
-| trust-store | DIRECTORY | "$HOME/.lcd"            | false    | directory for save checkpoints and validator sets    |
 
-Sample command:
-
-```bash
-gaiacli light-client --chain-id=test --laddr=tcp://localhost:1317  --node tcp://localhost:46657 --trust-node=false
+| Parameter       | Type      | Default                 | Required | Description                                          |
+| --------------- | --------- | ----------------------- | -------- | ---------------------------------------------------- |
+| chain-id        | string    | null                    | true     | ID of chain we connect to, must be specified |
+| home            | string    | "$HOME/.gaiacli"        | false    | directory for config and data, such as key and checkpoint |
+| node-list       | string    | "tcp://localhost:26657" | false    | Full node list to connect to, example: "tcp://10.10.10.10:26657,tcp://20.20.20.20:26657" |
+| laddr           | string    | "tcp://localhost:1317"  | false    | Address for server to listen on |
+| trust-node      | bool      | false                   | false    | Trust full nodes or not |
+| swagger-host-ip | string    | "localhost"             | false    | The host IP of the Gaia-lite server, swagger-ui will send request to this host |
+| modules         | string    | "general,key,token"     | false    | Enabled modules |
+
+Sample command to start gaia-lite node:
+```
+gaiacli lite-server --chain-id {your chain id}
```

Comment:
Lets change this for `gaiacli lite-server --chain-id=<chain_id>` to keep consistency with the other cmds